tests/integration/updateActivity.initTest.ts

```diff
@@ -0,0 +1,79 @@
+import { describe, Done } from "mocha";
```

Comment:
Can you please fix the name of the file to .intTest.ts?

Reply:
Am not aware of the Test file name conventions here in TS. Just followed as done by other functions. Will change for all.

Reply:
the problem is with the spelling, you named it updateActivity.**init**Test.ts instead of updateActivity.**int**Test.ts

Reply:
done.